src/signal.rs

```diff
@@ -395,6 +395,14 @@ impl Code {
     }
 }
 
+#[cfg(feature = "sbp-conversions")]
+impl std::convert::TryFrom<u8> for Code {
```

Comment:
Would it make sense to not gate this behind `sbp-conversions`? It doesn't actually use anything from `sbp`

Reply:
Good point. I expect basically all of the time that we will be converting integer values into `Code` will happen when dealing with SBP messages, but I guess there could be other situations where the integer to `Code` conversions could be needed. If we do that I think we should also add a conversion for `Constellation`.

Reply:
Changed!